lib/server.rb

```diff
@@ -16,6 +16,28 @@
 # To contact SUSE about this file by physical or electronic mail,
 # you may find current contact information at www.suse.com
 
+# Rack::TryStatic was taken from the rack-contrib repository (https://github.com/rack/rack-contrib)
+module Rack
+  class TryStatic
+    def initialize(app, options)
+      @app = app
+      @try = ['', *options[:try]]
+      @static = ::Rack::Static.new(lambda { |_| [404, {}, []] },options)
+    end
+
+    def call(env)
+      orig_path = env['PATH_INFO']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.